vapoursynth-sys/build.rs

```diff
@@ -43,12 +50,10 @@ fn main() {
 
 // Returns the default library dirs on Windows.
 // The default dir is where the VapourSynth installer puts the libraries.
-fn get_default_library_dir() -> Option<impl Iterator<Item = String>> {
-    let host = env::var("HOST").ok()?;
-
+fn get_default_windows_library_dir(target: String, host: String) -> Vec<String> {
```

Comment:
I guess this should work
```suggestion
fn get_default_windows_library_dir(target: &str, host: &str) -> Vec<String> {
```

Reply:
Good point